### DISCUSSION TITLE
REDIR远程解析的情况下，是否没有必要将远程DNS设置为DOT/DOH加密？

### DISCUSSION THREAD

**Discussion by Vegas7** [MetaCubeX/mihomo#1551] [Q&A]
如题，不知是否有加密的必要，因为实际解析发生在远程，唯一的疑问是将解析请求发给远程服务器的时候是否会被中间人看到？（如果被远程服务器看到那无所谓）

**Comment by refined-fish** [comment]
这个过程本地不发生解析，那是远程服务器的事情，你也控制不了吧
你对本地DNS服务器设置DoH只能让本地解析时加密罢了
